packages/config/config/devices/0x015f/mh-p220.json

```diff
@@ -13,57 +13,112 @@
 		{
 			"productType": "0x220a",
 			"productId": "0x5102"
+		},
+		{
+			"productType": "0x220a",
+			"productId": "0x5103"
 		}
 	],
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
 	},
-	"supportsZWavePlus": true,
 	"paramInformation": {
 		"1": {
-			"label": "Dimmer state after power down",
+			"label": "Restore State After Power Failure",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 1,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
 		},
 		"2": {
-			"label": "External switch type",
+			"label": "External Switch Type",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 2,
-			"defaultValue": 0,
+			"defaultValue": 1,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Momentary",
+					"value": 0
+				},
+				{
+					"label": "Switch",
+					"value": 1
+				},
+				{
+					"label": "Roller Blind Switch - S1 -> Up; S2 -> Down",
+					"value": 2
+				}
+			]
 		},
 		"3": {
-			"label": "2-state Switches",
+			"label": "Switch States",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 1,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Toggle",
+					"value": 0
+				},
+				{
+					"label": "On/off switch",
+					"value": 1
+				}
+			]
 		},
 		"4": {
-			"label": "ALL ON/ALL OFF",
+			"label": "All On / All Off",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 255,
 			"defaultValue": 255,
 			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable All Off; Disable All On",
+					"value": 1
+				},
+				{
+					"label": "Enable All On; Disable All Off",
+					"value": 2
+				},
+				{
+					"label": "Enable",
+					"value": 255
+				}
+			]
 		},
 		"5": {
-			"label": "Minimum brightness level",
+			"label": "Minimum Brightness",
 			"valueSize": 1,
 			"minValue": 1,
 			"maxValue": 98,
```

Comment:
Is 98 correct here? Not 99?

Reply:
Yes, 98 is correct - max brightness must be greater than minimum brightness.  So Minimum is 1-98, Max is 2-99

![image](https://user-images.githubusercontent.com/34757455/115202788-582ce380-a129-11eb-88a3-1a6662642a86.png)